src/app/controllers/sidemenuCtrl.js

```diff
@@ -18,23 +18,23 @@ function (angular, _, $, config) {
     $scope.setupMainNav = function() {
       $scope.mainLinks.push({
         text: "Dashboards",
-        icon: "fa fa-fw fa-th-large",
+        icon: "fa-fw icon-rt-dashboard",
```

Comment:
This is core grafana file, a change like this needs to be handled upstream in grafana, otherwise this will be a cause for merge issues.